### PR TITLE
Login with Users API

### PR DESCRIPTION
**Pull request by rsampdev**
Users are able to log in and log out, restructuring the file system for clarity, minor setup & exploration with listing the users active courses

### CODE REVIEW

**Comment by rsampdev** [comment]
> @googlebot I fixed it.

File: project/pom.xml [hunk 1]
```diff
@@ -119,6 +119,24 @@
             <artifactId>gson</artifactId>
             <version>2.8.6</version>
         </dependency>
+
+        <dependency>
```
Comment (BenLin00) [inline project/pom.xml:123]: duplicate dependency

**Review by BenLin00 — APPROVED** [review]
Very pretty code

File: project/pom.xml (hunk 1 above)
Comment (rsampdev) [reply project/pom.xml:123]: > Very pretty code

Thank you :)

> duplicate dependency

Thanks, nice catch, I'll merge and then fix this dependency in the next PR

**Pull request merged**